src/util/util_statuscode.h

```diff
@@ -5,20 +5,30 @@
 #include "util_log.h"
 
 namespace dxvk {
+    inline void Enter(const std::string& logMessage) {
+        log::write(str::format(logMessage, "|Enter"));
+    }
+
+    inline void Enter(const std::string& logMessage, bool omit) {
+        if (!omit) {
+            log::write(str::format(logMessage, "|Enter"));
+        }
+    }
```

Comment:
> `util_statuscode.h`

Is this a good place for that? It's not exactly related to status codes, although I'll admit we currently don't have anything that would be much better. Maybe just `util_log.h`?

Reply:
yeah, same thoughts, I was kinda lazy and not in the mood to introduce something new :)